docs/Migrating.md

```diff
@@ -13,10 +19,13 @@
      in the `Brain` GameObjects
   *  Agents have a `Brain` field in the Inspector, you need to drag the
   appropriate Brain asset in it.
-  
-__Note:__ You can pass the same brain to multiple agents in a scene by 
-leveraging Unity's prefab system or look for all the agents in a scene
-using the search bar of the `Hierarchy` window with the word `Agent`.
+* We modified the way we do inference with ML-Agents. All previous `.bytes`
```

Comment:
Replace with something like "We have changed the way ML-Agents models perform inference"